ebpf-profiler/README.md

```diff
@@ -0,0 +1,43 @@
+
+# Performance Profiler using BPF
+
+This is a basic example of how to get sample stack traces for the purposes of performance profiling, using eBPF.
```

Comment:
This is demo code to accompany the Pixie [blog post](https://blog.px.dev/cpu-profiling) showing how to get sample stack traces for the purposes of performance profiling, using eBPF.